lib/snyk.js

```diff
@@ -1,9 +1,10 @@
 'use strict'
 
 const request = require('request-promise-native')
+const { dressError } = require('./utils')
 
 const baseV1Url = 'https://snyk.io/api/v1'
-const baseRestUrl = 'https://api.snyk.io'
+const baseRestUrl = 'https://api.snyk.io/rest'
```

Comment:
I decided to include the full base here and just remove the '/rest' in the one place where it is redundant.

Reply:
@jeramysoucy I think this is now introducing this when used `Error: Failed to paginate request for get https://api.snyk.io/rest/rest/orgs/`, notice the double `/rest`

Reply:
Related issue: https://github.com/elastic/snyk-github-issue-creator/issues/30